weather_mv/loader_pipeline/bq.py

```diff
@@ -143,10 +146,23 @@ def to_table_schema(columns: t.List[t.Tuple[str, str]]) -> t.List[bigquery.Schem
     fields.append(bigquery.SchemaField(DATA_IMPORT_TIME_COLUMN, 'TIMESTAMP', mode='NULLABLE'))
     fields.append(bigquery.SchemaField(DATA_URI_COLUMN, 'STRING', mode='NULLABLE'))
     fields.append(bigquery.SchemaField(DATA_FIRST_STEP, 'TIMESTAMP', mode='NULLABLE'))
+    fields.append(bigquery.SchemaField(S2_LOCATION, 'GEOGRAPHY', mode='NULLABLE'))
 
     return fields
 
 
+def fetch_s2_location(lat, long):
+    """
+    This function calculates and returns s2_location
+    from the input latitude and longitude values
+    """
+    if lat > MAX_LATITUDE or lat < MIN_LATITUDE:
+        raise ValueError(f"Invalid latitude value '{lat}'")
+    long = ((long + 180) % 360) - 180
+    row = '{"type":"GeometryCollection","geometries":[{"type":"Point","coordinates":[%d,%d]}]}' % (long, lat)
```

Comment:
Please use python fstrings

Reply:
Should this be a string, or a dictionary? I thought in the documentation used GeoJSON via Python dictionaries?

Reply:
I don't think this should be a collection of a single point of coordinates. Rather, it should be a single point. Right?

If so, then the inner dictionary (starting with "type":"Point") should be sufficient.